### ISSUE TITLE
the browser smells to good

### ISSUE THREAD

**Issue by Thespikedballofdoom** [Pissandshittium/pissandshittium#148]
make it smelly oh god it hurts

**Comment by MacroHard-Winux** [comment]
Yeah, it makes my laptop make a strong perfume small but weirdly not my PC for some reason. Maybe it is because of the difference between the GPU? My laptop have intel-integrated graphics while my Gaming PC have GeForce RTX
3080 Ti. The bigger fan in my desktop GPU might help to dilute the small, but I am not sure

**Comment by Aikoyori** [comment]
we need access to ScentAPI
